sympy/stats/frv_types.py

```diff
@@ -88,7 +95,7 @@ def dict(self):
 
     @property
     def set(self):
-        return self.args
+        return set(self.args)
```

Comment:
isn't this supposed to be a SymPy set? I mean the `FiniteSet` class (is this method supposed to return a Python set?)

Reply:
I think,  `SingleFiniteDomain` converts the `set` and converts it to `FiniteSet`. Though I will confirm and get back to you.

Reply:
Yes, [L92](https://github.com/sympy/sympy/pull/16962/files#diff-4fdea159f969338cc732d53527a1977bR92) in `sympy.stats.frv` converts `set` to `FiniteSet` as follows,
```python
set = FiniteSet(*set)
```
So, I don't feel the need to pre-compute the trivial stuff.